src/OSPSuite.Core/Serialization/Xml/ModuleXmlSerializer.cs

```diff
@@ -13,5 +14,21 @@ public override void PerformMapping()
          Map(x => x.MergeBehavior);
          Map(x => x.IsPKSimModule);
       }
+
+      protected override void TypedDeserialize(Module objectToDeserialize, XElement outputToDeserialize, SerializationContext serializationContext)
+      {
+         base.TypedDeserialize(objectToDeserialize, outputToDeserialize, serializationContext);
+         objectToDeserialize.Snapshot = outputToDeserialize.Element("Snapshot")?.Value ?? string.Empty;
+      }
+
+      protected override XElement TypedSerialize(Module objectToSerialize, SerializationContext context)
+      {
+         base.TypedSerialize(objectToSerialize, context);
+         var element =  base.TypedSerialize(objectToSerialize, context);
+         var snapShot = new XElement("Snapshot", objectToSerialize.Snapshot);
+         element.Add(snapShot);
+
+         return element;
+      }
```

Comment:
Not sure here, When I add this as `Map(x => x.Snapshot)` then he whole base64 encoded string appears as an attribute.
This creates a separate element for it.

Reply:
you can specify in the mapping .AsNode or AsAttribute If my memories serves me whell

Reply:
also why do you care if it's an attribute or a node?

Reply:
Theoretically never, in practice, all the time.